### ISSUE TITLE
Not rotating local backups, remote yes

### ISSUE THREAD

**Issue by czPechy** [serghey-rodin/vesta#1615]
### Operating System (OS/VERSION):

CentOS 7.4.1708

### VestaCP Version:

0.9.8

### Installed Software (what you got with the installer):

nginx + apache
proftpd
exim + dovecot + spamassassin + clamav
iptables + fail2ban
file system quota - after install disabled

### Steps to Reproduce:

Hi, 
i have this issue on 3 same servers. 
All are Centos 7, all have updated Vesta.
From start i have all backups (local, remote - sftp) and everything works fine.
We storing 3-5 backups. (per machine - one has 5, others only 3)
After some months we have trouble with disk space on local machine, so i disable local backups (for 3-4 months)
And this year (approx. in Feb.) i enable local backups back. 
And from this enabling Vesta not deleting old backups, but only locally.
On remote machine (via sftp) are backups working fine.

So after one week we have fully disk.

**Comment by madeITBelgium** [comment]
Can you delete the old backups by your self and check later if it is still a problem?

**Comment by czPechy** [comment]
No, still same. For now i created little script for daily deleting old backups.

**Comment by Skamasle** [comment]
I cant reproduce this

Ensure your backups finish witout errors and your databases are fine try run backup from cli to check any error returned by script and try check databases with mysqlcheck

**Comment by czPechy** [comment]
I know that this will not be an easily identifiable problem.
Run from cli without error, still not rotating.
Mysqlcheck without problems.

Can not it cause remote backups to be turned on?

**Comment by Skamasle** [comment]
I cant reproduce this with remote or local backups, turn remote on, then off and still backups was rotated

**Comment by czPechy** [comment]
I found one little diference on this hosts. I have default backup folder "/backup" symlinked to "/home/backup". Maybe this is problem... But, as i said, cli not returns any errors.
